lib/push.py

```diff
@@ -113,9 +224,11 @@ def prepare(self):
          hash_uc = path_uc.file_hash()
          config["rootfs"]["diff_ids"].append("sha256:" + hash_uc)
          size_uc = path_uc.file_size()
-         path_c = path_uc.file_gzip(["-9", "--no-name"])
+         path_c_ = path_uc.file_gzip(["-9", "--no-name"])
```

Comment:
I'm a little confused by `path_uc` (no underscore) and `path_uc_` (with underscore). Maybe use more verbose names?